### ISSUE TITLE
sample does not work....

### ISSUE THREAD

**Issue by shinriyo** [florent37/Flutter-AssetsAudioPlayer#12]
sample does not work....
I just build sample in `/sample`.

I pushed button but `log: resource not` log appeared.
```
flutter: log: resource not found song1.mp3
flutter: log: openOptional({
    file = "song2.mp3";
    folder = "assets/audios";
})
flutter: log: resource not found song2.mp3
flutter: log: openOptional({
    file = "song3.mp3";
    folder = "assets/audios";
})
flutter: log: resource not found song3.mp3
flutter: log: playnil
flutter: log: openOptional({
    file = "song2.mp3";
    folder = "assets/audios";
})
flutter: log: resource not found song2.mp3
flutter: log: openOptional({
    file = "song2.mp3";
    folder = "assets/audios";
})
```

**Comment by shinriyo** [comment]
I read the blog

https://medium.com/flutter-community/how-to-add-music-audio-to-your-flutter-app-dcb6162c32d7

|Package|	iOS	|Android|
|assets_audio_player|	✖|	✓|

**Comment by florent37** [comment]
fixed on 1.0.2
